pageserver/src/repository.rs

```diff
@@ -746,8 +743,8 @@ mod tests {
 
         let mut lsn = 0x10;
         for blknum in 0..pg_constants::RELSEG_SIZE + 1 {
-            let img = TEST_IMG(&format!("foo blk {} at {}", blknum, Lsn(lsn)));
             lsn += 0x10;
+            let img = TEST_IMG(&format!("foo blk {} at {}", blknum, Lsn(lsn)));
```

Comment:
The LSN in "foo blk {} at {}" string that is stored in the content of the page versions was wrong, because we advanced 'lsn' after creating the string. That was unrelated to this PR really, I just happened to spot it while working on this.

I opened a separate PR for that: https://github.com/zenithdb/zenith/pull/1315